bot/events/ready.ts

```diff
@@ -0,0 +1,23 @@
+import { botCache } from "../cache.ts";
+import { getApplicationInformation } from "../deps.ts";
+
+/** Event handler for the READY event when bot is fully online. */
+export async function ready(clusterID: number) {
+  // TODO: Change to a logger  like winston. Deno has a logger built in and many deps as well.
+  console.log(`Client Connected | Cluster ${clusterID}`);
+
+  const application = await getApplicationInformation().catch(() =>
+    // TODO: Use logger.debug
+    console.log("Failed to load owners from application")
+  );
+  if (!application) return;
+
+  // ADDS ALL TEAM MEMBERS AS OWNERS
+  if (application.team)
+    application.team.members.forEach((member) => {
+      botCache.ownerIDs.add(member.user.id!);
+    });
+
+  // ADDS THE OWNER THEMSELF AS OWNER. THE SET WILL REMOVE IF REDUNDANT FROM TEAM.MEMBERS
+  botCache.ownerIDs.add(application.owner.id!);
```

Comment:
Shouldn't this be an if/else statement? If application.team then do X else do Y.

Reply:
Technically, yes you are right but in essence this is the same thing but imo this is cleaner without if else. I prefer avoiding nested code whenever possible. Performance wise this is basically nill as well as sets/maps are insanely fasst 90/mil ops/s and this is 1 extra op. This isn't worth nesting imo.

If you still prefer if/else let me know and i can change